README.md

```diff
@@ -19,13 +19,13 @@ Square, Google, and other contributors.
 
 ## Status
 
-  - ***Release Version:* 2.2**
-  - ***Snapshot Version:* 2.3-SNAPSHOT**
+  - ***Release Version:* 2.5**
+  - ***Snapshot Version:* 2.5-SNAPSHOT**
 
 Dagger is currently in active development, primarily internally at Google,
 with regular pushes to the open-source community. Snapshot releases are
 auto-deployed to sonatype's central maven repository on a clean build with
-the version `2.3-SNAPSHOT`.
+the version `2.5-SNAPSHOT`.
```

Comment:
:) yeah, I'm sure that'll be fixed sometime soon, in which this will probably wanted to get updated again.